src/System.Private.ServiceModel/tests/Scenarios/SelfHostWcfService/WcfService.csproj

```diff
@@ -5,19 +5,19 @@
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProjectGuid>{CBE9909F-4CF7-4AD0-9307-BD0DFBB1105E}</ProjectGuid>
-    <OutputType>Exe</OutputType>
+    <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>WcfService</RootNamespace>
     <AssemblyName>WcfService</AssemblyName>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
+    <OutputPath>..\Bridge\bin\WcfService\</OutputPath>
```

Comment:
It would be best if any build artifacts and build output were placed into the bin/ directory rather than anywhere under src/.